server.go

```diff
@@ -46,6 +47,11 @@ func NewServer(c *structs.Config) (*Server, error) {
 	pc := controllers.NewPeopleController(session, c)
 	getRouter.HandleFunc("/people/{id:[0-9]+}/", pc.Get)
 
+	staticC := controllers.NewStaticController(c)
+	getRouter.HandleFunc("/about/", staticC.GetAbout)
+	getRouter.HandleFunc("/contact/", staticC.GetContact)
+	getRouter.HandleFunc("/getinvolved/", staticC.GetInvolved)
```

Comment:
Yeah, I kinda agree with you, but I'd have no idea on how to make that happen myself.